router-old.yaml

```diff
@@ -0,0 +1,3 @@
+supergraph:
```

Comment:
@BrynCooke Was this file intentionally added? It doesn't seem to be referenced anywhere.

Reply:
Thanks for the spot.
https://github.com/apollographql/router/pull/2750